doc/source/NewSiteConfigs.rst

```diff
@@ -249,7 +249,7 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
    # Check your clang version then add it to your site compiler config.
    clang --version
    spack config add "packages:all:compiler:[apple-clang@YOUR-VERSION]"
-   spack config add "packages:all:providers:mpi:[openmpi@4.1.6]"
+   spack config add "packages:all:providers:mpi:[openmpi@5.0.1]"
```

Comment:
This looks like a good update too. Perhaps this will help with the MacOS build issue #971?

Reply:
maybe